cmd/api-errors.go

```diff
@@ -830,10 +833,25 @@ var errorCodes = errorCodeMap{
 		Description:    "The replication target does not exist",
 		HTTPStatusCode: http.StatusNotFound,
 	},
+	ErrBucketRemoteRemoveDisallowed: {
+		Code:           "BucketRemoteRemoveDisallowed",
```

Comment:
These errors should be XMinio prefixed @poornas

Reply:
done